unstructured/ingest/connector/fsspec.py

```diff
@@ -111,13 +127,76 @@ def get_file(self):
             **self.config.get_access_kwargs(),
         )
         logger.debug(f"Fetching {self} - PID: {os.getpid()}")
-        fs.get(rpath=self.remote_file_path, lpath=self._tmp_download_file().as_posix())
+        try:
+            fs.get(rpath=self.remote_file_path, lpath=self._tmp_download_file().as_posix())
+        except FileNotFoundError as e:
+            logger.error("Failed to find file in remote source.")
+            logger.error(e)
```

Comment:
hmm, what is the value of catching here? would we not want this to raise? if we can't get the file we definitely don't want to move to the next step of trying to partition the downloaded file.

Reply:
when you pull in from latest you'll find that we added a wrapper this so that errors raised in get_file will be wrapped as a [SourceConnectionError](https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/ingest/error.py#L28) with the wrapping decorator like[ this](https://github.com/Unstructured-IO/unstructured/blob/92692ad8d7d5001601dd88fef869a29660f492cb/unstructured/ingest/connector/fsspec.py#L105).

Reply:
This will mean that all errors raised in this method will get a wrapped with a log `Error in connecting to upstream data source: `

Reply:
which is why I think we can do nothing here (i.e. don't try/catch just let it raise the exception)

Reply:
which is why I think we can do nothing here (i.e. don't try/catch just let it raise the exception)